sentry-android-integration-tests/sentry-uitest-android/src/androidTest/java/io/sentry/uitest/android/AutomaticSpansTest.kt

```diff
@@ -90,7 +90,7 @@ class AutomaticSpansTest : BaseUiTest() {
                 // AGP matrix tests have no frames
                 Assume.assumeTrue(totalFrames > 0)
                 assertNotEquals(totalFrames, 0)
-                assertTrue(totalFrames > slowFrames + frozenFrames, "Expected total frames ($totalFrames) to be higher than the sum of slow ($slowFrames) and frozen ($frozenFrames) frames.")
+                assertTrue(totalFrames >= slowFrames + frozenFrames, "Expected total frames ($totalFrames) to be higher than the sum of slow ($slowFrames) and frozen ($frozenFrames) frames.")
```

Comment:
```suggestion
                assertTrue(totalFrames > slowFrames + frozenFrames, "Expected total frames ($totalFrames) to be higher than the sum of slow ($slowFrames) and frozen ($frozenFrames) frames.")
```
This test checks there is at least 1 normal frame that is summed to slow + frozen frames. That's why the test should be "long" enough to have at least 1 normal frame even on slower devices
Otherwise total frames is always at least slow + frozen, so it wouldn't make sense to have it, and it could be removed directly